lib/generators/rad_common/install/install_generator.rb

```diff
@@ -342,6 +343,17 @@ def rad_config_setting_exists?(setting_name)
           File.readlines(RAD_CONFIG_FILE).grep(/#{setting_name}:/).any?
         end
 
+        def remove_legacy_rails_config_setting
+          return unless File.exist?(RAD_CONFIG_FILE)
```

Comment:
this should always exist so it shouldn't be needed, we'd probably want it to crash if it's not there since something weird would be happening